dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/utils/FeatureInfo.java

```diff
@@ -178,6 +184,7 @@ private synchronized void populateInfo() {
             if (vis != null) {
                 visibility = vis.trim();
             }
+            this.shortName = builder.getProperty("IBM-ShortName");
```

Comment:
This should use the `FeatureBnd.IBM_SHORT_NAME` constant... I can't see why any of those constants should be private? A little pointless?

Reply:
These constants seem to be duplicated in multiple places. Perhaps `com.ibm.ws.kernel.feature.internal.subsystem.FeatureDefinitionUtils` should be the definitive source?

Reply:
I switched to use the FeatureBnd constant.